packages/mdc-menu/simple/foundation.js

```diff
@@ -37,6 +37,7 @@ export default class MDCSimpleMenuFoundation extends MDCFoundation {
       removeClass: (/* className: string */) => {},
       hasClass: (/* className: string */) => {},
       hasNecessaryDom: () => /* boolean */ false,
+      eventTargetHasClass: (/* target: EventTarget, className: string */) => false,
```

Comment:
+1 to @amsheehan's comment. The method is opinionless by design, because we [don't want to make reference's to anything DOM-related within an foundation code](https://github.com/material-components/material-components-web/blob/master/docs/authoring-components.md#do-not-reference-host-objects-within-foundation-code), the primary place where an adapter is used.